blockade/tests/test_chaos.py

```diff
@@ -0,0 +1,81 @@
+import time
+import unittest
+
+from mock import patch, MagicMock
+
+from blockade import errors
+from blockade import chaos
+from blockade.api.manager import BlockadeManager
+
+
+class ChaosTest(unittest.TestCase):
+
+    def setUp(self):
+        self.chaos = chaos.Chaos()
+
+    def tearDown(self):
+        block_mock = MagicMock()
+        with patch.object(BlockadeManager, 'get_blockade',
+                          return_value=block_mock):
+            self.chaos.shutdown()
+
+    def test_basic_start_stop_destroy(self):
+        name = "aname"
+        block_mock = MagicMock()
+        with patch.object(BlockadeManager, 'get_blockade',
+                          return_value=block_mock):
+            self.chaos.new_chaos(name)
+            self.chaos.pause(name)
+            self.chaos.delete(name)
```

Comment:
Is there anything we can assert here to make sure new, pause, and delete did something (besides just not throw an exception)?